htdocs/accountancy/admin/account.php

```diff
@@ -25,7 +25,7 @@
  */
 
 // Load Dolibarr environment
-require '../../main.inc.php';
+require_once __DIR__.'/../../main.inc.php';
```

Comment:
the main.inc.php must never be included several times, it is the init of env that must be done once and only once.
So we must use the require and not require_once

Reply:
I don't understand what you're saying.
>the main.inc.php must never be included several times SO we use require instead of require_once

Isn't that the very purpose of require_once ?

Reply:
Hi, @eldy, 
>A page should never include another page (this will generates conflict in mime type, duplicate http header, content policies rules, and more evil side effects.

Have you encounter these problems or is that a general assumption ? Is that not dependant on how you code your page ?

As I finally succeed to make work with hooks, can you confirm me that is the way I search ?

Did you happen to see the modifications that i've done inside the templates of modulebuilder ? I find this more elegant. Will that be interesting to include that in a specific PR ?